browser/src/control/Control.Menubar.js

```diff
@@ -565,7 +566,8 @@ L.Control.Menubar = L.Control.extend({
 					{name: _('Dark Mode'), id: 'toggledarktheme', type: 'action'},
 					{uno: '.uno:SidebarDeck.PropertyDeck', name: _UNO('.uno:Sidebar')},
 					{uno: '.uno:Navigator', id: 'navigator'},
-					{name: _('Show Status Bar'), id: 'showstatusbar', type: 'action'}
+					{name: _('Show Status Bar'), id: 'showstatusbar', type: 'action'},
+					{name: _('Collapse Notebook Bar'), id: 'collapsenotebookbar', type: 'action'},
```

Comment:
For compact view we shouldn't add collapsenotebookbar menu item because we are now in the compact view.